### DISCUSSION TITLE
Move specific language LSP documentation away from github wiki, into docs

### DISCUSSION THREAD

**Discussion by dessalines** [helix-editor/helix#6540] [General]
The language documentation for setting up specific LSPs is currently in the github wiki, rather than the official documentation.

https://docs.helix-editor.com/languages.html

It would be better to move these into the official docs, and either close the github wiki, or just link to the official docs. Currently, it wasn't easy to find these since they are in two places.

**Comment by David-Else** [comment]
I suggested that, but the main devs are not keen on the idea. I still think it would be better as now the majority of languages have been added and the whole thing needs working on as a whole.

I started to clean up the WIKI article yesterday, but there is a load more work to do on that. I thought some language logos might be nice, especially in SVG, but they all need batch intelligently resizing.

**Comment by pascalkuthe** [comment]
We don't really want to maintain language-specific information. That puts even more review burden on the maintainers. The wiki functions more like a Wikipedia with anyone able to edit without our review. Anything that is related to helix (but not about helix itself) like configuration for external tools fits there.

The official documentation on the website is only intended for official documentation about helix itself. Altough we have discussed adding more prominent links to the wiki there (like adding dummy pages with the content "this page is maintained by out community in our wiki")

**Reply by dessalines** [reply]
The main issue is that the first and most important question in adding any new editor is: does it support my language? So I went to the helix docs, and it had no documentation for any of the languages I use. 

It seemed like a lot of people used helix, so I had to do some more googling, but none of it revealed the wiki page. Not sure how I managed to stumble upon the wiki page luckily, because it isn't linked anywhere. 

Most importantly, these are arguably the 2nd most important docs for installing helix, and its stored only in githubs proprietary wiki system. Any other code mirrors or backups won't have this very important info. 

Once the bulk of the languages are added (and that work already looks done), it shouldn't be a too much of a burden to merge docs PRs. You can see that it's only changing that file, and isn't dangerous to merge.

You could also separate the docs reviewers in various ways to keep them separate from code reviews. 

Having a page that shows you how to set up helix for some of the more popular languages could potetially increase adoption too.
